Homework_12/Task_2.py

```diff
@@ -0,0 +1,14 @@
+
+def own_min(*args):
+    sorted_args = sorted(args)
+    return f'The min number is: {sorted_args[0]}'
```

Comment:
I am not sure that this function should return a string. Better return sorted_args[0]

Reply:
Got it🫡